packages/welcome/lib/welcome-view.js

```diff
@@ -111,7 +110,7 @@ export default class WelcomeView {
               <li>
                 The {this.brand} forum at{' '}
                 <a
-                  href="https://github.com/pulsar-edit/pulsar/discussions"
+                  href="https://github.com/orgs/pulsar-edit/discussions"
```

Comment:
```suggestion
                  href="https://github.com/orgs/pulsar-edit/discussions"
```

Personally still feel like this should link to our ORG wide discussion. Which if I recall correctly, is using the Pulsar discussions

Reply:
I'm not sure I understand, that suggested change is the same as what I've put isn't it?
![image](https://user-images.githubusercontent.com/58074586/202326024-97c4efc7-3c98-45ba-8672-7fa4a5227394.png)

Reply:
Apparently I've been staring at a screen for to long today. My bad